src/Particle/Lattice/LRBreakupParameters.h

```diff
@@ -34,9 +34,11 @@ class LRBreakupParameters<T, 3>
   ///number of strictly enforced periodic spatial dimensions
   /// ewald_strict2d sets ndim=2, otherwise ndim=3
   unsigned ndim;
+  unsigned ewaldGrid;
```

Comment:
In the source code, we better to use a clear name for the variable. "ewald_num_grid_points". Also please add a line of documentation.